devU-api/src/entities/grader/grader.serializer.ts

```diff
@@ -0,0 +1,9 @@
+import { SubmissionScore, SubmissionProblemScore, GraderInfo } from 'devu-shared-modules'
+
+export function serialize(scores: any[]): GraderInfo {
+  const submissionScore = scores.pop()
+  return {
+    submissionScore: submissionScore as SubmissionScore,
+    submissionProblemScores: scores as SubmissionProblemScore[]
```

Comment:
-Needs further discussion-

I don't like this at all. If I'm reading things right, your pushing the submission score as the first element of the array and the rest of the array is the problem scores. This needs to be cleaner and clearer. 

SubmissionScore and SubmissionProblemScore already have serializers. If anyone wants those values, they should hit the existing endpoints and use that logic. 

The real question is, what should the grading endpoint return? Should it return anything besides an ACK?.. Grading will take a while when we start using containers. The best thing might be to immediately return something like "got your request and we're starting grading"